docs/editor/github.md

```diff
@@ -19,6 +19,8 @@ Once you've installed the [GitHub Pull Requests and Issues](https://marketplace.
 
 ![Extension Sign In](images/github/extension-signin.png)
 
+Note that there are several ways to authenticate to GitHub, including using your username and password with two-factor authentication (2FA), a personal access token, or an SSH key. See [About authentication to GitHub](https://docs.github.com/en/github/authenticating-to-github/about-authentication-to-github) for more information and details about each option. 
```

Comment:
I'm not sure if this is useful to include here. While this is true, when logging in through VSCode, we always open the browser and have the user login there.

Reply:
Once they get to the browser, does the username and password always work? The feedback we received was around providing more information about SSH and other ways to authenticate. 

Is there another doc where it makes sense to include this reference link?

Reply:
It sometimes fails on Linux, which isn't able to reopen VSCode afterwards to complete the process. But it should allow 2FA.

Maybe this would make sense to move below to the change about authentication not working automatically.